static/js/deadline-handler.js

```diff
@@ -0,0 +1,23 @@
+const DEADLINE_DATE = CONSTANTS.DEADLINE_DATE;
+const deadlineSecs = Math.floor(Date.parse(DEADLINE_DATE) / 1000);
+
+const deadlineCheck = () => {
+    ETA = deadlineSecs - Math.floor(Date.now() / 1000);
+    if ((ETA) <= 0) {
+        $('.countdown-text').css({
+            "display": "none"
+        })
+        $('.apply-btn').css({
+            "display": "none"
+        })
+        $('.navbar-register').css({
+            "display": "none"
+        })
```

Comment:
I think we should not hide the buttons after "time runs out" since the client's date-time might be slightly inaccurate. When we actually close the apps, we should merge a PR to remove the button.

Also, the nav button should always remain as that's the page where people RSVP or edit their application